cmd/certsuite/run/run.go

```diff
@@ -111,13 +111,13 @@ func runTestSuite(cmd *cobra.Command, _ []string) error {
 
 	testParams := configuration.GetTestParameters()
 	if testParams.ServerMode {
-		log.Info("Running CNF Certification Suite in web server mode")
+		log.Info("Running Workload Certification Suite in web server mode")
```

Comment:
`Workload Certification Suite` --> `Certification Suite`. Some for the other changes.